src/Swarm/App.hs

```diff
@@ -79,9 +78,11 @@ appMain port mseed scenario toRun cheat = do
       gsRef <- newIORef (s ^. gameState)
       eport <- Swarm.Web.startWebThread port gsRef
 
-      let s' = case eport of
-            Left e -> s & runtimeState . eventLog %~ (|> LogEntry 0 ErrorTrace "Web API" (-2) zero (T.pack e))
-            Right p -> s & runtimeState . webPort ?~ p
+      let s' =
+            s & runtimeState
+              %~ case eport of
+                Right p -> webPort ?~ p
```

Comment:
Should we generate an informational message here saying that the web server was successfully started on port nnn?

Reply:
Done!

![image](https://user-images.githubusercontent.com/44544735/188485086-44124604-b712-496c-a278-491790adaec9.png)